Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM teamserverless/license-check:0.3.9 as license-check
```

Comment:
This also can be updated now to https://github.com/openfaas/license-check/pkgs/container/license-check and 0.4.0